src/quantization.cpp

```diff
@@ -489,12 +492,11 @@ void quantize_int8(program& prog,
 
 // For the input of each input argument, we need to insert a
 // capture operator to compute the scale and shift
-std::size_t capture_arguments(program& prog,
-                              const std::vector<std::string>& ins_names,
-                              const std::function<void(std::size_t, std::vector<argument>)>& func)
+void capture_arguments(module& m,
+                       const std::vector<std::string>& ins_names,
+                       const std::function<void(std::size_t, std::vector<argument>)>& func,
+                       std::size_t& num_quant_params)
```

Comment:
Whats the purpose of this? Wont we know this based on the value of the last capture op?